springboard/templates/base.jinja2

```diff
@@ -14,17 +14,19 @@
     <div id="header">
         <div class="lang">
             <div>
-                {% for code, name in languages %}
-                <span>
-                    <a href="/locale/?language={{code}}">{{name}}</a>
-                    {% if not forloop.last %}
-                        <span> |</span>
-                    {% endif %}
-                </span>
+                <li>
+                    <a>{{language|display_language_name}}</a>
+                </li>
+                {% for lang_code in display_languages %}
+                <li>
```

Comment:
That's taken care of here https://github.com/universalcore/springboard/blob/feature/issue-21-add-locale-change/springboard/views.py#L55 in order to avoid duplication, but I'll put a check in the template, just to be sure